### PR TITLE
prep-javascript-variables - Jeremy Nguyen

### PR DESCRIPTION
**Pull request by jeremyhnguyen**
Screenshot 2023-06-17 at 1 17 12 PM

### CODE REVIEW

File: prep-javascript-variables/index.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      console.log(fullName);
+      console.log(typeof fullName);
+      const numberOfPets = 1;
+      console.log(numberOfPets);
+      console.log(typeof numberOfPets);
+      const likesVegetables = true;
+      console.log(likesVegetables);
+      console.log(typeof likesVegetables);
+      const nothing = null;
+      const never = console.log(nothing);
```
Comment (BrettAlbright) [inline prep-javascript-variables/index.html]: you want to use `let` here so you can not define it a value. Const requires a value at declaration and you want a variable with an undefined value here.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
one small update needed

File: prep-javascript-variables/index.html (hunk 1 above)
Comment (jeremyhnguyen) [reply prep-javascript-variables/index.html]: Screenshot 2023-06-19 at 2 07 35 PM

File: prep-javascript-variables/index.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      console.log(typeof fullName);
+      const numberOfPets = 1;
+      console.log(numberOfPets);
+      console.log(typeof numberOfPets);
+      const likesVegetables = true;
+      console.log(likesVegetables);
+      console.log(typeof likesVegetables);
+      const nothing = null;
+      // eslint-disable-next-line prefer-const
+      let never = console.log(nothing);
```
Comment (BrettAlbright) [inline prep-javascript-variables/index.html]: this should be 
let never;

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
small change on the let variable needed.

File: prep-javascript-variables/index.html (hunk 2 above)
Comment (jeremyhnguyen) [reply prep-javascript-variables/index.html]: Screenshot 2023-06-20 at 8 15 47 PM

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**